hub/management/commands/import_new_constituencies.py

```diff
@@ -17,9 +17,9 @@ class Command(BaseCommand):
 
     """
     This uses the geopackage from
-      https://pages.mysociety.org/2025-constituencies/datasets/parliament_con_2025/latest
+      https://pages.mysociety.org/2025-constituencies/datasets/parliament_con_2024/latest
```

Comment:
This feels like this shouldn't be in, as presumably mySociety have retained the paths upstream.

Reply:
Correct  - changing